presto-product-tests/conf/presto/etc/jvm.config

```diff
@@ -19,3 +19,4 @@
 -Duser.timezone=UTC
 -Xdebug 
 -Xrunjdwp:transport=dt_socket,server=y,suspend=n,address=5005
+-XX:ErrorFile=/docker/volumes/logs/product-tests-presto-jvm-error-file.log
```

Comment:
No, we want the additional file that the JVM produces. It defaults to being in /var/presto/hs_err_pid5.log, and is lost when the docker container stops (which happens immediately after Presto terminates).